lib/matplotlib/_constrained_layout.py

```diff
@@ -672,6 +669,7 @@ def reposition_colorbar(layoutgrids, cbax, renderer, *, offset=None):
 
     Parameters
     ----------
+    layoutgrids :
```

Comment:
```suggestion
    layoutgrids : dict
```
Also add the renderer below?